AE-1247: investigate monitoring large positions that may be unhealthy close to liquidation (#15)
arbitraryexecution/compound-monitoring: liquidation-monitor/README.md

# Compound Liquidation Monitor

## Description

> This agent detects when an account on Compound is able to be liquidated.

Reviewer: ```suggestion
This bot detects when an account on Compound is able to be liquidated.
```

## Optimizations

> - Although there are ~400k accounts using Compound, limit the search to 1 wei ETH borrowed it will reduce the tracked accounts down to just over 9000. A nearly 98% reduction in assets being tracked.

Reviewer: ```suggestion
- Although there are ~400k accounts using Compound, limiting the search to 1 wei ETH borrowed reduces the tracked accounts down to just over 9000. A nearly 98% reduction in assets being tracked.
```

- Lowering the maxHealth setting will further reduce this list.

> - Initial account information can be obtained from Compound API

Reviewer: ```suggestion
- Initial account information can be obtained from Compound API.
```
